index.bs

```diff
@@ -144,6 +128,17 @@ partial interface DocumentOrShadowRoot {
 	</dd>
 </dl>
 
-Applying Styles In All Contexts {#styles-in-all-contexts}
-===================
+Every {{DocumentOrShadowRoot}} has adopted stylesheets (of type FrozenArray&lt;CSSStyleSheet>).
```

Comment:
Use `<dfn>` around "adopted stylesheets" here, then when referring to it later, say `[=adopted stylesheets=]` to link to it.